### PR TITLE
[#130] Minor fixes and improvements following serialization branch merge

### PR DESCRIPTION
**Pull request by evwhite29**
Main updates:
- Fixed link issue in serialized user payload
- Finalized whitelist of publicly available endpoints (no auth required)
- Cleanup controller action route comments and callbacks
- Add new lat and lng attributes to organization model, complete with seed generation and serialization.

### CODE REVIEW

File: app/controllers/api/v1/organizations_controller.rb [hunk 1]
```diff
@@ -1,22 +1,19 @@
 module Api::V1
   class OrganizationsController < ApiBaseController
-    before_action :set_organization, only: [:show, :update, :grant_permission, :admins, :destroy]
```
Comment (dan-crayne) [inline app/controllers/api/v1/organizations_controller.rb:3]: Does this remove the ability to delete organizations? If so, then what is our reasoning for doing this?

Comment (evwhite29) [reply app/controllers/api/v1/organizations_controller.rb:3]: We decided that we weren't going to allow organizations or programs to be deleted at the last Monday meeting. That action is unreachable due to how the routes are set up anyway. For our own capabilities, we can just use the console if we really want to delete an organization or program, but if you think there is a case to leave this open for the super_admin, let me know. If so, the routes would have to be updated as well. But there are no plans to utilize this action from the FE at this time.

Comment (dan-crayne) [reply app/controllers/api/v1/organizations_controller.rb:3]: I might have misunderstood the final decision during the meeting. It still seems like this basic functionality should be supported, but if it won't be available in the FE, then it probably makes sense to leave it out.

**Review by dan-crayne — APPROVED** [review]
Looks good!

File: app/controllers/api/v1/organizations_controller.rb (hunk 1 above)
Comment (evwhite29) [reply app/controllers/api/v1/organizations_controller.rb:3]: Well, the only way to really "support it" is to have a fully functioning route and action for it, at which point we're back to making the same decision about what to do with associated programs when an organization is deleted.

**Review by iandusenbury — APPROVED** [review]

**Pull request merged**